src/Package/MSBuild.UnGAC.Setup/NativeMethods.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Build.UnGAC
+{
+    [ComImport, Guid("E707DCDE-D1CD-11D2-BAB9-00C04F8ECEAE"), InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    internal interface IAssemblyCache
+    {
+        [PreserveSig]
+        uint UninstallAssembly(uint dwFlags, [MarshalAs(UnmanagedType.LPWStr)] string pszAssemblyName, IntPtr refData, int pulDisposition);
```

Comment:
```suggestion
        uint UninstallAssembly(uint dwFlags, [MarshalAs(UnmanagedType.LPWStr)] string pszAssemblyName, IntPtr refData, ref int pulDisposition);
```

? it's defined as `[out, optional] ULONG *pulDisposition`. Though since we're still doing best-effort we don't need to check it.

Reply:
extreme nit: could you be more consistent about dropping or not dropping the Hungarian prefixes?